src/components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,39 @@
+import "./ProductCard.css";
+import { Link } from "react-router-dom";
+const ProductCard = ({ product }) => {
+  return (
+    <div className="card card-vt">
+      <div className="card-header">
+        <Link to="/">
+          <img className="card-img" src={product.img} alt={product.title} />
+        </Link>
+        <span className="btn-wishlist">
+          <i className="fas fa-heart" />
+        </span>
+      </div>
+
+      <div className="card-text-content">
+        <div className="card-title">
+          <p>{product.title}</p>
+          <span className="card-bdg">
```

Comment:
classNames could be better

Reply:
I have defined this class in the component library, will change it there first then implement it in this project